static/app/components/modals/demoEndModal.tsx

```diff
@@ -0,0 +1,176 @@
+import {css} from '@emotion/react';
+import styled from '@emotion/styled';
+
+import {fetchGuides} from 'sentry/actionCreators/guides';
+import {ModalRenderProps} from 'sentry/actionCreators/modal';
+import {navigateTo} from 'sentry/actionCreators/navigation';
+import Button from 'sentry/components/button';
+import ModalTask from 'sentry/components/onboardingWizard/modalTask';
+import {SidebarPanelKey} from 'sentry/components/sidebar/types';
+import {IconClose} from 'sentry/icons/iconClose';
+import {t} from 'sentry/locale';
+import SidebarPanelStore from 'sentry/stores/sidebarPanelStore';
+import {Organization} from 'sentry/types';
+import useApi from 'sentry/utils/useApi';
+import {useRouteContext} from 'sentry/utils/useRouteContext';
+
+// tour is a string that tells which tour the user is completing in the walkthrough
+type Props = ModalRenderProps & {orgSlug: Organization['slug'] | null; tour: string};
+
+export default function DemoEndingModal({tour, closeModal, CloseButton, orgSlug}: Props) {
+  const api = useApi();
+
+  const {router} = useRouteContext();
+
+  let cardTitle = '',
+    body = '',
+    guides = [''],
+    location = '';
+
+  switch (tour) {
+    case 'issues':
+      cardTitle = t('Issues Tour');
+      body = t(
+        'Thank you for completing the Issues tour. Learn about other Sentry features by starting another tour.'
+      );
+      guides = ['issues_v3', 'issue_stream_v3'];
+      location = `/organizations/${orgSlug}/issues/`;
+      break;
+    case 'performance':
+      cardTitle = t('Performance Tour');
+      body = t(
+        'Thank you for completing the Performance tour. Learn about other Sentry features by starting another tour.'
+      );
+      guides = ['performance', 'transaction_summary', 'transaction_details_v2'];
+      location = `/organizations/${orgSlug}/performance/`;
+      break;
+    case 'releases':
+      cardTitle = t('Releases Tour');
+      body = t(
+        'Thank you for completing the Releases tour. Learn about other Sentry features by starting another tour.'
+      );
+      guides = ['releases_v2', 'react-native-release', 'release-details_v2'];
+      location = `/organizations/${orgSlug}/releases/`;
+      break;
+    case 'tabs':
+      cardTitle = t('Check out the different tabs');
+      body = t(
+        'Thank you for checking out the different tabs. Learn about other Sentry features by starting another tour.'
+      );
+      guides = ['sidebar_v2'];
+      location = `/organizations/${orgSlug}/projects/`;
+      break;
+    default:
+  }
+
+  const sandboxData = window.SandboxData;
+  const url = sandboxData?.cta?.url || 'https://sentry.io/signup/';
+
+  async function handleRestart() {
+    await Promise.all(
+      guides.map(guide =>
+        api.requestPromise('/assistant/', {
+          method: 'PUT',
+          data: {guide, status: 'restart'},
+        })
+      )
+    );
+
+    if (tour === 'issues') {
+      localStorage.setItem('issueGuide', '1');
```

Comment:
Yea I think this use case is more suited to a store/global state rather than local storage. working with @roggenkemper to see if we can use a reflux store here.